app/views/jobseekers/job_applications/qualifications/fields/_secondary_school.html.slim

```diff
@@ -1,9 +1,16 @@
-= f.govuk_fieldset legend: { text: t("helpers.legend.jobseekers_job_application_details_qualifications_shared_legends.subjects") } do
+= f.govuk_fieldset legend: { text: t("helpers.legend.jobseekers_job_application_details_qualifications_shared_legends.subjects") }, class: "subjects-and-grades" do
   - form.qualification_results.each_with_index do |result, idx|
     = f.fields_for :qualification_results, result do |result_form|
-      .inline-fields-container id="qualification-result-#{idx}"
+      .inline-fields-container.subject-row id="qualification-result-#{idx}" class="#{idx > form.highest_present_result_index ? "js-hidden" : ""}"
         = result_form.govuk_text_field :subject, label: { text: "Subject #{idx + 1}" }, aria: { required: idx.zero? }, form_group: { classes: "govuk-!-width-two-thirds" }
         = result_form.govuk_text_field :grade, aria: { required: idx.zero? }, form_group: { classes: "govuk-!-width-one-third" }
+        .govuk-form-group.button_to
```

Comment:
This delete button only exists for rows after the first row - `if idx.positive?`

Reply:
gonna talk with craig about this, i feel there are problems if we do this...

Reply:
If we are going with a delete button for every row, then the `aria-required`s might need to change too.

Reply:
i think aria-required should still be true for the first row wether there is a delete button or not